packages/server/lib/controllers/runner.ts

```diff
@@ -92,8 +92,7 @@ export const runner = {
   },
 
   handle (testingType, req: Request, res: Response) {
-    // FIXME: Why is runner-ct being used for all injections? Can anyone comment here as to why this changed?
-    const pathToFile = getPathToDist(testingType === 'e2e' ? 'runner' : 'runner-ct', req.params[0])
+    const pathToFile = getPathToDist('runner', req.params[0])
```

Comment:
Nice, I wonder if we can just serve this from `runner-shared` instead? Ideally we should eventually eliminate both `runner` and `runner-ct` entirely.

In this case you'd probably need to change the webpack config to write the `dist` to `runner-shared`, or even move it there entirely. Thoughts?

Reply:
I think we need to move towards removing the runner dirs all together. Ideally the app package would handle setting this up, even if both vite & webpack need to be used to bundle everything correctly. 🤷🏻‍♀️

Reply:
Good point, there isn't really a compelling reason for `runner*` to exist, we may as well just move all the things to `app` (including webpack deps, config, etc). More packages just adds more misdirection.

Reply:
When I read the current code, 

* `runner-ct` was dependent on `runner-shared` to use `dom` and `stduio`. 
* `runner` was dependent on `runner-ct` to use `unified-runner`.

So, the workflow became:

1. Migrate the `runner-shared` to `runner-ct`. (Remove unnecessary studio react files and some scss)
2. Migrate the `runner-ct` to `runner`. (Remove unnecessary scss files)

Now all the necessary codes of `runner-ct` and `runner-shared` are in `runner`.